### PR TITLE
Issue 441 - Delete marathon app instances of logstash and db if they exist

### LINKED ISSUE #441: Re-deploy logstash and database
The current Scale deployment assumes that the scale-logstash application has not been deployed, which is an incorrect assumption. Update the deployElk.py script to destroy the scale-logstash application if it already exists before deploying it. Also destroy and deploy the database if DEPLOY_DB is true.

### PR DESCRIPTION
**Pull request by tclarke**
closes #441 

Always delete logstash. If the user requests db initialization, delete postgis as well.

### CODE REVIEW

File: dockerfiles/framework/scale/deployDb.py [hunk 1]
```diff
@@ -25,8 +28,7 @@ def run():
     marathon = '''{"id": "/%(scaleDBHost)s",
                  "instances": 1,
                  "constraints": [["hostname","UNIQUE"]],
-                 "container": {"docker": {"forcePullImage": true,
```
Comment (gisjedi) [inline dockerfiles/framework/scale/deployDb.py]: Is there a specific motivation for removing forcePullImage?

File: dockerfiles/framework/scale/deployElk.py [hunk 2]
```diff
@@ -25,8 +28,7 @@ def run():
       "container": {
         "docker": {
           "image": "%s",
-          "network": "HOST",
-          "forcePullImage": true
```
Comment (gisjedi) [inline dockerfiles/framework/scale/deployElk.py]: Same comment on forcePullImage as above.

**Comment by gisjedi** [comment]
Commit https://github.com/ngageoint/scale/pull/468/commits/597b097a145d9c217a250dbe0f3cd4ecaab8d1a0 has been cherry-picked into https://github.com/ngageoint/scale/pull/470 pull request for testing, so I'm closing this pull request. If more changes are needed here we can re-open.

File: dockerfiles/framework/scale/deployDb.py (hunk 1 above)
Comment (tclarke) [inline dockerfiles/framework/scale/deployDb.py]: Keeping it there slows down deployment by ignoring the local image cache. It needs to pull the image again every time you startup.

**Pull request closed without merging**